SwissTransferResources/Localizable/de.lproj/Localizable.strings

```diff
@@ -3,22 +3,34 @@
  * Project: SwissTransfer
  * Locale: de, German
  * Tagged: ios
- * Exported by: Valentin Perignon
- * Exported at: Thu, 25 Jul 2024 09:20:03 +0200
+ * Exported by: Ambroise Decouttere
+ * Exported at: Thu, 25 Jul 2024 14:57:40 +0200
  */
 
 /* loco:66a1fa770af8cf8470002102 */
 "contentDescriptionCreateNewTransferButton" = "Create New Transfer";
 
+/* loco:66a24a3f3472a720090cfd52 */
+"downloadedTransferLabel" = "Downloaded transfer: %d/%d ";
+
 /* loco:669f7a817f40f7e9170fafa2 */
 "expiresIn" = "Expires in %d days";
 
 /* loco:669f76984c6fad682900d032 */
 "firstTransferDescription" = "Make your first transfer!";
 
+/* loco:66a24aee8d282979b007c5d2 */
+"fromHeader" = "From:";
+
+/* loco:66a24acfa855c577100e2353 */
+"messageHeader" = "Message:";
+
 /* loco:669f759d2093b5a83708e5e2 */
 "receivedTitle" = "Received";
 
+/* loco:66a24aaccbb152954307a612 */
+"recipientsHeader" = "Recipient(s):";
```

Comment:
I guess we will have a singular and plural strings. We should create both, and only use the singular for the moment